react-commenting.js

```diff
@@ -112,13 +122,15 @@ class Square extends Component{
 
   handleSquareClick = () => {
     // 12) Here:
+    //This check for the winner on each click to update the board.
     this.props.gamePlay(this.props.index)
   }
 
   render(){
     return(
       <div id="square" onClick={ this.handleSquareClick }>
         {/* 13) Here: */}
+        //This is the parent of rendered squares index.
         { this.props.value }
```

Comment:
This is displaying the current value of the square - meaning it starts as blank then if the square is clicked it updates the value to X or O.